bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -65,6 +73,7 @@ spec:
         - apiGroups:
           - ""
           resources:
+          - namespaces
           - pods
```

Comment:
Why is the bundle manifest committed?

Reply:
This needs to be generated and committed based on what Andrew and Gurney told us.  It is not generated during downstream build (just select fields are replaced, like version)

Reply:
it already was.
The only change is when I added the check if ACM/MCE already exists I added a role in the installer (probably that role was already present as it was working) but I didn't regenerate that file.